[Delegation Toolkit] Advanced Permissions improvements (#2438)
MetaMask/metamask-docs: delegation-toolkit/guides/advanced-permissions/execute-on-metamask-users-behalf.md

### 4. Check the EOA account code

Currently, Advanced Permissions (ERC-7115) does not support automatically upgrading a MetaMask user's account to a [MetaMask smart account](../../concepts/smart-accounts.md). Therefore, you must 

> ensure that the user is upgraded to a smart account before requesting Advanced Permissions (ERC-7115).

Reviewer: ```suggestion
Currently, Advanced Permissions do not support automatically upgrading a MetaMask user's account to a [MetaMask smart account](../../concepts/smart-accounts.md). Therefore, you must 
ensure that the user is upgraded to a smart account before requesting Advanced Permissions.
```

}
```

> ### 5. Request Advanced Permissions (ERC-7115)

Reviewer: ```suggestion
### 5. Request Advanced Permissions
```

</Tabs>

> ### 7. Redeem Advanced Permissions (ERC-7115)

Reviewer: ```suggestion
### 7. Redeem Advanced Permissions
```
